src/utils/accessibility/accessibility.js

```diff
@@ -3,20 +3,32 @@ function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize,fontUnitOfMeasure) {
```

Comment:
There is a minor styling issue here. Usually we put space after the comma. 
Could you run `npm run lint` to and clean the issues reported? (you'll see a few more)